doc/man3/SSL_CIPHER_get_name.pod

```diff
@@ -88,6 +92,13 @@ TLS 1.3 cipher suites) B<NID_auth_any> is returned. Examples (not comprehensive)
 SSL_CIPHER_is_aead() returns 1 if the cipher B<c> is AEAD (e.g. GCM or
 ChaCha20/Poly1305), and 0 if it is not AEAD.
 
+SSL_CIPHER_find() returns a B<SSL_CIPHER> structure which has the cipher ID B<ptr>.
```

Comment:
Hmm, the only routine I see that takes SSL_CIPHER input is SSL_CIPHER_get_id(), also in this PR, that gives the 0x0300-prefixed form.
But, SSL_early_get0_ciphers() will get you wire-format ciphersuite representations from inside an early callback.